themes/shijin4/layouts/partials/footer.html

```diff
@@ -2,5 +2,6 @@
 	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
 
 	<script src="/js/navbar.js"></script>
+	<script src="/js/asciinema-player.js"></script>
```

Comment:
This file is probably pretty large; please only include it when it's actually needed. Same for the CSS.